src/components/index.js

```diff
@@ -6,5 +6,16 @@ import { ProductItem } from './ProductItem';
 import { ProductList } from './ProductList';
 import { SortingOptions } from './SortingOptions';
 
+import { BookTypeFilterOption } from './FilterOptions/BookTypeFilters/BookTypeFilterOption'
+import { BookTypeFilters } from './FilterOptions/BookTypeFilters/BookTypeFilters';
```

Comment:
you don't need to import it you can directly write
```
export { BookTypeFilters } from './FilterOptions/BookTypeFilters/BookTypeFilters';
```
Then you won't need the export in the end

Reply:
Oh yes, makes sense. I've made the changes.